.circleci/config.yml

```diff
@@ -565,13 +577,85 @@ jobs:
       - run:
           name: Build the plugin
           command: |
+            mkdir -p /tmp/artifacts
             yarn build
             npx grunt clean:build copy:build
+            if [ "$CIRCLE_JOB" != 'master' ]; then
+              COBLOCKS_VERSION_LINE=$(awk '/\* Version:/' ./build/coblocks/class-coblocks.php)
+              VER_NUMBER=$(echo "$COBLOCKS_VERSION_LINE" | tr -dc '0-9.')
+              COBLOCKS_COMMIT_HASH=$(git rev-parse --verify HEAD | head -c 7)
+              sed -i -e "s/${VER_NUMBER}/${VER_NUMBER}-${CIRCLE_PULL_REQUEST##*/}+${COBLOCKS_COMMIT_HASH}/g" build/coblocks/class-coblocks.php
+            fi
+            cd build
+            if [ "$CIRCLE_JOB" == 'master' ]; then
+              zip -r "coblocks-v${VER_NUMBER}.zip" coblocks
+            else
+              zip -r "coblocks-${CIRCLE_PULL_REQUEST##*/}.zip" coblocks
+            fi
+            mv ~/project/build/*.zip /tmp/artifacts
+      - store_artifacts:
+          path: /tmp/artifacts
       - persist_to_workspace:
           root: ~/project
           paths:
             - build
             - dist
+            - build-num.txt
+
+  pr_artifact_comment:
+    docker:
+      - image: circleci/php:7.3-node
```

Comment:
@EvanHerman can we bump that to at least 7.4?

Reply:
👍 Good call